docs/api.rst

```diff
@@ -1373,8 +1373,11 @@ Querying all balances
 
    :reqjson bool async_query: Boolean denoting whether this is an asynchronous query or not
    :reqjson bool ignore_cache: Boolean denoting whether to ignore the cache for this query or not.
+   :reqjson bool save_data: Boolean denoting whether to force save data even if the balance save frequency has not lapsed (see `Getting or modifying settings`_ ).
```

Comment:
you can actually provide a hyperlink to that section here in rst. See how it's done for `_balances_result`

Reply:
Done.

Reply:
For all those done comments, did you push to github yet? If yes then something is wrong with Github and I can't see it.

Reply:
No, sorry for the confusion, I will do so after I try to fix the test as well.